.github/workflows/codeql.yml

```
@@ -0,0 +1,62 @@
+name: CodeQL
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)